cli/pcluster/cli_commands/update.py

```diff
@@ -54,19 +54,27 @@ def execute(args):
         cfn_client = boto3.client("cloudformation")
         _restore_cfn_only_params(cfn_client, args, cfn_params, stack_name, target_config)
 
+        s3_bucket_name = cfn_params["ResourcesS3Bucket"]
```

Comment:
Apparently, S3 get object action is already there (because of HIT) https://docs.aws.amazon.com/parallelcluster/latest/ug/iam.html#parallelclusteruserpolicy